realm/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Wed Dec 13 12:08:46 GMT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.3.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions-snapshots/gradle-4.4-20171031235950+0000-all.zip
```

Comment:
The official version of Gradle 4.4 is already released.

And we should not edit `gradle-wrapper.properties` directly. See https://github.com/realm/realm-java/blob/master/README.md#upgrading-gradle-wrappers